v3/src/components/slider/slider-model.ts

```diff
@@ -9,6 +9,9 @@ export const SliderModel = TileContentModel
   .props({
     type: types.optional(types.literal(kSliderTileType), kSliderTileType),
     multipleOf: 0.5,
+    animationRate: 1,
+    direction: types.string,
+    repetition: types.string,
```

Comment:
We should do something like that. I'm going to be back in this area shortly since I'm working on the `GlobalValueModel` and `SliderModel` and so I can give it some thought then. It might make sense to just adopt the v2 compatible numeric options and use named constants.